src/screenipy.py

```diff
@@ -89,35 +68,117 @@ def initExecution():
             raise ValueError
         return result
     except:
-        print(colorText.BOLD + colorText.FAIL + '\n[+] Please enter a valid numeric option & Try Again!' + colorText.END)
+        print(colorText.BOLD + colorText.FAIL +
+              '\n[+] Please enter a valid numeric option & Try Again!' + colorText.END)
         sleep(2)
         Utility.tools.clearScreen()
         return initExecution()
 
+
+def screenStocks(executeOption, reversalOption, daysForLowestVolume, minRSI, maxRSI, respBullBear, insideBarToLookback, stock):
```

Comment:
`screenStocks` should take `testing` as a boolean parameter to work properly with `pytest`.
* Run all methods from Screener
* if `testing==True` then stop the process **`if len(screeningDictionary)`**  as we don't want to screen all the stock for testing. All we need is to get 1 stock which is a result of test parameters that we have set during testing.

Reply:
ok, I will work on it.

Reply:
Things that I tried: 
I replaced ```starmap``` to ```starmap_async``` and created while loop to check len(screenResults) >= 1 and on suceess I terminating ```pool```.  But calling ```pool.terminate()``` deletes all processes of pool and other queued stock symbols gets stuck.

Another thing that I tried is in ```screenStocks``` method used ```testing==True && len(screenResults) >= 1``` and on success I am calling sys.exit() which delete that process. But due to same reasons as above because of the pending stocks symbols, it gets stuck.

Can we check for only the first 100  or 200 stocks symbols when ```testing == True```  ? 
We definitely going to get more than 1 stock in screenResults for 100 or 200 stocks.

Reply:
* Is there anyway to safely terminate the pool even if there are pending symbols?
* Can we get the process IDs and pass them to process itself so the process can terminate itself `if testing==True and len(listStockCodes)` 
* Also maybe same issue of pending symbols happening with `KeyboardInterrupt`